boefjes/boefjes/job_handler.py

```diff
@@ -209,15 +209,26 @@ def handle(self, normalizer_meta: NormalizerMeta) -> None:
             results = self.job_runner.run(normalizer_meta, raw)
             connector = self.octopoes_factory(normalizer_meta.raw_data.boefje_meta.organization)
 
+            logger.info("Obtained results %s", str(results))
+
             for observation in results.observations:
+                for result in observation.results:
+                    if self._parse_ooi(result).primary_key == observation.input_ooi:
```

Comment:
We are doing self._parse_ooi tree times here. lets store the results in a var, and reuse the data.

Reply:
Addressed in d0f04700de0211e53ed572da84b08dcddc67cce5.